regulations/settings/base.py

```diff
@@ -87,55 +76,54 @@
 STATICFILES_FINDERS = (
     'django.contrib.staticfiles.finders.FileSystemFinder',
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
-#    'django.contrib.staticfiles.finders.DefaultStorageFinder',
 )
 
 SECRET_KEY = os.environ.get('DJANGO_SECRET_KEY', get_random_string(50))
 
-# List of callables that know how to import templates from various sources.
-TEMPLATE_LOADERS = (
-    ('django.template.loaders.cached.Loader', (
-    'django.template.loaders.filesystem.Loader',
-    'django.template.loaders.app_directories.Loader',)),
-#     'django.template.loaders.eggs.Loader',
-)
+TEMPLATES = [{
+    "BACKEND": "django.template.backends.django.DjangoTemplates",
+    "OPTIONS": {
+        "context_processors": (
+            # "django.contrib.auth.context_processors.auth",
+            "django.template.context_processors.debug",
+            "django.template.context_processors.i18n",
+            "django.template.context_processors.media",
+            "django.template.context_processors.static",
+            "django.template.context_processors.tz",
+            "django.contrib.messages.context_processors.messages"
+        ),
+        # List of callables that know how to import templates from various
+        # sources.
+        "loaders": [
+            ('django.template.loaders.cached.Loader', (
```

Comment:
I agree that it's unclear -- the config isn't easy to read. From https://docs.djangoproject.com/en/1.8/ref/templates/api/

> **loaders** is a list of template loader classes, specified as strings. Each Loader class knows how to import templates from a particular source. Optionally, a tuple can be used instead of a string. The first item in the tuple should be the Loader class name, subsequent items are passed to the Loader during initialization.

We're using the `cached` loader, which requires an argument defining which loaders it should cache. This is equivalent to how `TEMPLATE_LOADERS` worked, save that it's now confined to a specific template system